src/ec/packages/ec-component-footer-harmonised/footer-harmonised.html.twig

```diff
@@ -0,0 +1,98 @@
+{% spaceless %}
+
+{#
+  Parameters:
+  - "group" (string) (default: 'group1'): 'group1' or 'group2'
+  - "sections" (array of objects) (default: []):
+    - "title" (optional) (string) OR (object with Link component in property)
+    - "title_class_name" (optional) (string) (default: '')
+    - "description" (optional) (string) (default: '')
+    - "content_before" (optional) (string) (default: '')
+    - "list_class_name" (optional) (string) (default: '')
+    - "links" (optional) (array of Link components) (default: []):
+  - "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the icon
+  - "extra_attributes" (optional) (array) (default: []) Extra attributes for icon
+    - "name" (string) Attribute name, eg. 'data-test'
+    - "value" (string) Attribute value, eg: 'data-test-1'
+#}
+
+{# Internal properties #}
+
+{% set _group = group|default('group1') %}
+{% set _sections = sections|default([]) %}
+{% set _css_class = 'ecl-footer-harmonised ecl-footer-harmonised--' ~ _group %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% if attr.value is defined %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value|e ~ '"' %}
+    {% else %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name %}
+    {% endif %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<footer class="{{ _css_class }}"{{ _extra_attributes|raw }}>
```

Comment:
we are adding a space between classes and attributes recently, i would adapt all the components to this "standard"..

Reply:
ok